src/__tests__/to-contain-html.js

```diff
@@ -1,7 +1,7 @@
 import {render} from './helpers/test-utils'
 
 /* eslint-disable max-statements */
-test('.toContainHTML', () => {
+describe('.toContainHTML', () => {
```

Comment:
This change makes no sense. Code inside `describe` blocks is not run as a test. You are effectively disabling all our past tests here.

What you need to do is to either wrap all the previously existing tests inside a new `test` or `it` block inside the `describe`. This block will stand alongside the new `test` block you added.

Reply:
Read and understood the differences between `describe`, `test` and `it`. Made the changes